packages/babel-plugin-transform-bigint/src/index.js

```diff
@@ -0,0 +1,41 @@
+import syntaxBigInt from "babel-plugin-syntax-bigint";
+
+export default function({ types: t }) {
+  return {
+    inherits: syntaxBigInt,
+
+    visitor: {
+      Program: {
+        exit(path: NodePath, state: Object) {
+          state.file.addImport("bigint-polyfill", "default", "BigInt");
```

Comment:
This is fine for now, we also use this for transform-runtime. I wouldn't call it a "polyfill" if you are just going to use the variable (then it's just the same as transform-runtime) rather than babel-polyfill.